security-council-rehearsals/2024-01-24-pause-mainnet/README.md

```diff
@@ -0,0 +1,215 @@
+
+Execution can be finalized by anyone once a threshold of signatures
+are collected, so a Facilitator will do the final execution for
+convenience.
+
+## Approving the transaction
+
+### 1. Install tools
+
+```
+just install
+```
+
+### 2. Setup Ledger
+
+Your Ledger needs to be connected and unlocked. The Ethereum
+application needs to be opened on Ledger with the message “Application
+is ready”.
+
+Check your address is correct by running the following command
+
+```
+eip712sign --address --hd-paths "m/44'/60'/0'/0/0" --ledger
+```
+
+You should see the output similar to the following:
+
+```
+Signer: 0x8C835568fE7Eea01B6DCD79214aB5BCe5E1759B0
+``` 
+
+If this is not the address you are expecting,
+you can change the `hd-paths` parameter, i.e. `--hd-paths "m/44'/60'/1'/0/0"`.
+
+### 3. Sign the transactions
+
+We currently have 5 transactions to be signed inside the `tx` folder:
+- `draft-86.json`
+- `draft-unpause-87.json`
+- `draft-88.json`
+- `draft-89.json`
+- `draft-90.json`
+
+Copy them to `~/presigner/tx` folder.
+
+You can sign them one by one by running the following commands:
+
+```
+cd ~/presigner
+
+presigner \
+        --json-file tx/draft-86.json \
+        --ledger \
+        --hd-paths "m/44'/60'/0'/0/0" \
+        sign
+
+presigner \
+        --json-file tx/draft-unpause-87.json \
+        --ledger \
+        --hd-paths "m/44'/60'/0'/0/0" \
+        sign
+
+presigner \
+        --json-file tx/draft-88.json \
+        --ledger \
+        --hd-paths "m/44'/60'/0'/0/0" \
+        sign
+
+presigner \
+        --json-file tx/draft-89.json \
+        --ledger \
+        --hd-paths "m/44'/60'/0'/0/0" \
+        sign
+
+presigner \
+        --json-file tx/draft-90.json \
+        --ledger \
+        --hd-paths "m/44'/60'/0'/0/0" \
+        sign
+```
+
+For each transaction  we will be performing 3 validations
+and ensure the domain hash and  message hash are the same
+between the Tenderly simulation and your
+Ledger:
+
+1. Validate integrity of the simulation.
+2. Validate correctness of the state diff.
+3. Validate and extract domain hash and message hash to approve.
+
+#### 3.1. Validate integrity of the simulation.
+
+Make sure you are on the "Overview" tab of the tenderly simulation, to
+validate integrity of the simulation, we need to
+
+1. "Network": Check the network is Ethereum Mainnet.
+2. "Timestamp": Check the simulation is performed on a block with a
+   recent timestamp (i.e. close to when you run the script).
+3. "Sender": Check the address shown is your signer account. If not,
+   you will need to determine which “number” it is in the list of
+   addresses on your ledger. 
+
+Here is an example screenshot, note that the Timestamp and Sender
+might be different in your simulation:
+
+![](./images/tenderly-overview-network.png)
+
+#### 3.2. Validate correctness of the state diff.
+
+Now click on the "State" tab. Verify that:
+
+1. Under address `0xbEb5Fc579115071764c7423A4f12eDde41f106Ed` (OptimismPortal contract),
```

Comment:
Should this still be for the OptimismPortal, or should it target the SuperchainConfig?